src/multi-select-item/multi-select-item.component.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes } from 'react';
+import { Checkbox } from 'react-bootstrap';
+import './multi-select-item.component.scss';
+
+export default class MultiSelectItem extends React.PureComponent {
```

Comment:
For some reason, PureComponent doesn't seem to implement shouldComponentUpdate even for a primitive type as boolean. I tested it with a performance lib, it does re-renders everything when the check of isChecked change is removed